README.md

```diff
@@ -179,9 +179,9 @@ Once you have created a cluster, you can interact with the cluster and machine
 resources using kubectl:
 
 ```bash
-kubectl --kubeconfig=kubeconfig get clusters
-kubectl --kubeconfig=kubeconfig get machines
-kubectl --kubeconfig=kubeconfig get machines -o yaml
+kubectl --kubeconfig=$KUBECONFIG get clusters
```

Comment:
After kind set `KUBECONFIG`, we actually do not need `--kubeconfig=$KUBECONFIG`.

Here the command is a generic command for KIND, MiniKube and existing cluster, can you help think more to update the document work for all of those three cases?